examples/flutter_gallery/.idea/runConfigurations/main_dart.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="flutter_gallery" type="FlutterRunConfigurationType" factoryName="Flutter">
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter">
```

Comment:
We should keep the name of the launch config the same - to say which module its for. It would be difficult to select a particular one if they all say `main.dart`.